### DISCUSSION TITLE
Track mileage for vehicles without an odometer by trip distance

### DISCUSSION THREAD

**Discussion by ack100** [hargata/lubelog#442] [Ideas]
A way to enter the trip distance for vehicles/trailers without an odometer would be handy.

I think a separate box for distance input or having the odometer box detect a plus symbol (+) and increment the odometer.

Manually adding the trip distance to the last odometer number works for now.

Awesome software

**Comment by hargata** [comment]
I have started to look into this. I think how I will approach this is to add a button right next to the Odometer field with the "+" icon, when clicked on it, you will be prompted to enter a number which will then be added to the last reported odometer reading for that vehicle. I have created #449 to track this.

**Comment by contrerasp** [comment]
I hope that the option to auto calculate the distance is very convenient, although i can definitely see why you would want it changed. Would it be possible to make it similar to how the MPG calculations are optional?

**Comment by hargata** [comment]
Released in 1.3.0
